tensorflow/compiler/mlir/tosa/transforms/legalize_utils.cc

```diff
@@ -565,6 +630,26 @@ LogicalResult getVectorFromValue32(Value val, SmallVectorImpl<int32_t>& vec) {
   return success();
 }
 
+// Populate a int64_t vector from a val tensor
+// return failure if val is not a constant value
+// return success otherwise
+LogicalResult getVectorFromValue64(Value val, SmallVectorImpl<int64_t>& vec) {
+  int i = 0;
+
+  ElementsAttr elems;
+
+  vec.clear();
+
+  if (!matchPattern(val, m_Constant(&elems))) return failure();
+
+  for (auto idx : elems.getValues<IntegerAttr>()) {
+    vec.push_back(static_cast<int64_t>(idx.getInt()));
+    i++;
```

Comment:
What does i do?

Reply:
good catch, seems useless, will take a deeper look and update in future patches.